docs/installation.rst

```diff
@@ -90,23 +81,27 @@ Now, we describe how to install HistomicsML using docker container.
           "IPAddress": "",
           "IPAddress": "192.80.0.1",
 
-7. Modify IP address in ``account.php`` on ``histomicsml_hmlweb_1`` container:
+6. Modify IP address in ``account.php`` in the web container
 
+ * Switch to the web docker container
 .. code-block:: bash
-
  $ docker exec -t -i histomicsml_hmlweb_1 bash
  root@97d439b58033:/# cd /var/www/html/HistomicsML/db
  root@97d439b58033:/var/www/html/HistomicsML/db# vi account.php
+ 
+ * Create the account.php file in your text editor directing the database to the address from step 5
+.. code-block:: bash
+ $ cat > account.php
+ $ dbAddress = "192.80.0.2"; => $dbAddress = "192.80.0.1"
 
- * Open up the account.php in your text editor and modify $dbAddress.
- * $dbAddress = "192.80.0.2"; => $dbAddress = "192.80.0.1"
-
-8. Start learning server:
+7. Start the server
 
 .. code-block:: bash
 
  root@97d439b58033:/var/www/html/HistomicsML/db# service al_server start
  Starting active learning server daemon al_server [ OK ]
  root@97d439b58033:/var/www/html/HistomicsML/db# exit
+ 
+.. note:: If the server becomes unresponsive or generates a connection error during use, the al_server will need to be restarted.
```

Comment:
@cooperlab In order to start the al_server again, 1. stop the al_server using "service al_server stop" 2. wait for a few seconds for the system ready 3. start the al_server using "service al_server start".